app/javascript/mastodon/components/column.jsx

```diff
@@ -3,6 +3,8 @@ import PropTypes from 'prop-types';
 import { supportsPassiveEvents } from 'detect-passive-events';
 import { scrollTop } from '../scroll';
 
+const listenerOptions = supportsPassiveEvents ? { passive: true } : true;
```

Comment:
Shouldn't it use `capture: true` in both cases?
```suggestion
const listenerOptions = supportsPassiveEvents ? { passive: true, capture: true } : true;
```

That being said, I'm not sure this class needs changes at all.

Reply:
I chose to move the inline ones to `listenerOptions` here, to be similar to the other files (and its ran only one time).

The mistake here is that when there is no passive events support, the argument should be `false` (or `undefined`, `false` is the default value), which is consistent with the object arg.